convert2rhel/toolopts.py

```diff
@@ -63,6 +80,7 @@ def _get_argparser():
             "  convert2rhel [-h]\n"
             "  convert2rhel [--version]\n"
             "  convert2rhel [-u username] [-p password | -f pswd_file] [--pool pool_id | -a] [--disablerepo repoid]"
+            "  convert2rhel [-c | --config-file conf_file]"
             " [--enablerepo repoid] [--serverurl url] [--keep-rhsm] [--no-rpm-va] [--debug] [--restart]"
             " [--disable-colors] [-y]\n"
```

Comment:
You changed:
`convert2rhel [-u username] [-p password | -f pswd_file] [--pool pool_id | -a] [--disablerepo repoid] [--enablerepo repoid] [--serverurl url] [--keep-rhsm] [--no-rpm-va] [--debug] [--restart] [--disable-colors] [-y]`
to
`convert2rhel [-u username] [-p password | -f pswd_file] [--pool pool_id | -a] [--disablerepo repoid]  convert2rhel [-c | --config-file conf_file] [--enablerepo repoid] [--serverurl url] [--keep-rhsm] [--no-rpm-va] [--debug] [--restart] [--disable-colors] [-y]`

If somebody runs the command this way, convert2rhel would fail.

As I understand the meaning of pipe it's that you should use either one option or another, not both. In this case, for example, either password or a config file.

It should be (including removing the deprecated `-f pswd_file`):
```python
            "  convert2rhel [-u username] [-p password | -c conf_file_path] [--pool pool_id | -a] [--disablerepo repoid]"
            " [--enablerepo repoid] [--serverurl url] [--keep-rhsm] [--no-rpm-va] [--debug] [--restart]"
            " [--disable-colors] [-y]\n"
            "  convert2rhel [--no-rhsm] [--disablerepo repoid]"
            " [--enablerepo repoid] [--no-rpm-va] [--debug] [--restart] [--disable-colors] [-y]\n"
            "  convert2rhel [-k activation_key | -c conf_file_path] [-o organization] [--pool pool_id | -a] [--disablerepo repoid] [--enablerepo"
            " repoid] [--serverurl url] [--keep-rhsm] [--no-rpm-va] [--debug] [--restart] [--disable-colors] [-y]"
```

Reply:
Thanks for clarification what the pipe does :+1: